PUBBLICATI/Articolo-CommonVoice.md

```diff
@@ -1,90 +1,76 @@
 
 ### Parla
 
 ![](https://user-images.githubusercontent.com/403283/42370763-e332a15c-810d-11e8-8863-ad82b806576e.png)
 
-Da questa interfaccia sarà possibile registrare un file audio leggendo il testo che è visualizzato. Sarà possibile riascoltare la registrazione ed approvarla caricandola.
+Tramite la scheda "**Parla**" sarà possibile registrare un file audio leggendo il testo visualizzato. Sarà possibile riascoltare la registrazione e approvarla prima di confermarne il caricamento.
 
 ### Ascolta
 
 ![](https://user-images.githubusercontent.com/403283/42370811-078ce1f2-810e-11e8-9a62-7eb41b91e8e7.png)
 
-Da questa interfaccia invece si potranno ascoltare le registrazioni fatte da altri e poterle approvare o rifiutare, ad esempio non è stata pronunciata correttamente una parola o è troppo veloce. Questa attività è ideale da fare quando non si può parlare ma si può ascoltare (ad esempio in cuffia).
+Tramite la scheda "**Ascolta**" sarà possibile ascoltare le registrazioni fatte da altri utenti, così da approvarle o rifiutarle, quest'ultimo caso è utile nell'eventualità il termine (o l'intera frase) non venga detta in maniera corretta, oppure se sia stata pronunciata troppo velocemente. Si tratta di un'attività ideale da portare a termine quando si è impossibilitati a parlare ma non ad ascoltare (magari durante una visita in biblioteca, con le cuffie sulle orecchie).
 
 ### Scorciatoie
 
-Ci sono anche le scorciatoie da tastiera per non dover utilizzare il mouse, ad esempio *P* per mettere in pausa o ascoltare o *S* per saltare la registrazione.
-
-## Quando sarà disponibile il modello Italiano
-
-Il piano di Mozilla è di rilasciare i modelli per ogni lingua verso Dicembre ma questo sarà possibile solo se ci sono tante registrazioni e tanti partecipanti. La qualità del riconsocimento vocale sta a noi!  
+Alcune scorciatoie da tastiera ti permetteranno di muoverti nell'interfaccia più rapidamente. premi *P* per mettere in pausa (o riprendere l'ascolto) o *S* per saltare alla successiva registrazione.
 
-## Voglio promuovere il progetto, come posso fare?
+## Quando sarà disponibile il modello Italiano?
 
-Fantastico! Se utilizzi Telegram vieni a conoscere la comunità su https://t.me/mozItaHUB, specialmente sul canale Voglio diventare un volontario.  
-Troverai membri della comunità italiana che potranno rispondere alle tue domande e fornirti idee per promuovere il progetto.  
+Il piano di Mozilla è di rilasciare i modelli di ogni lingua intorno al termine dell'anno (Dicembre), ma questo sarà possibile solo se saranno state effettuare tante registrazioni grazie alla partecipazioni di altrettanti volontari. La qualità del riconoscimento vocale sta a noi!  
 
-Il lancio in Italiano è corrisposto con un evento a Torino:
+## Voglio promuovere il progetto, come posso farlo?
 
-https://twitter.com/astrastefania/status/1014929254612598784
```

Comment:
pensavo di lasciarlo perché è il primo evento dedicato a common voice in italia

Reply:
È nel passato, non serve rimembrare nulla (anche perché è un tweet con una fotografia, nulla più, nessun resoconto, niente di niente).

Reply:
questo é vero allora direi che va bene per essere pubblicato capo!